core/trino-main/src/test/java/io/trino/execution/scheduler/TestBinPackingNodeAllocator.java

```diff
@@ -686,24 +686,24 @@ public void testAllocateSpeculative()
             acquireSpeculative3.release();
 
             // we have 32G free on NODE_1 now
-            NodeAllocator.NodeLease acquireNonSpeculative4 = nodeAllocator.acquire(REQ_NONE, DataSize.of(32, GIGABYTE), STANDARD);
-            assertAcquired(acquireNonSpeculative4, NODE_1);
+            NodeAllocator.NodeLease acquireStandard4 = nodeAllocator.acquire(REQ_NONE, DataSize.of(32, GIGABYTE), STANDARD);
+            assertAcquired(acquireStandard4, NODE_1);
 
             // no place for speculative task
             NodeAllocator.NodeLease acquireSpeculative4 = nodeAllocator.acquire(REQ_NONE, DataSize.of(1, GIGABYTE), SPECULATIVE);
             assertNotAcquired(acquireSpeculative4);
 
             // no place for another non-speculative task
-            NodeAllocator.NodeLease acquireNonSpeculative5 = nodeAllocator.acquire(REQ_NONE, DataSize.of(32, GIGABYTE), STANDARD);
-            assertNotAcquired(acquireNonSpeculative5);
+            NodeAllocator.NodeLease acquireStandard5 = nodeAllocator.acquire(REQ_NONE, DataSize.of(32, GIGABYTE), STANDARD);
```

Comment:
1,2,3,4,5 -- could the variable names be more descrptive?

or maybe with some helper method we could avoid variable allocations at all (less chance of asserting on a wrong variable if there are no local variables)

Reply:
It is hard. We need variables as they hold state we are assessing (future), somtimes multiple times.
As for naming I am not sure I can do much better.